src/js/DrawStyleController.js

```diff
@@ -9,7 +9,148 @@ goog.require('ga_styles_service');
   ]);
 
   module.controller('GaDrawStyleController', function($scope, $translate,
-      gaStyleFactory) {
+      gaGlobalOptions, gaStyleFactory) {
+
+    var iconsCSS = [
```

Comment:
name it iconsCatgeory0 . You could potientially have more than 1 categorie with css icons.